src/main/java/org/jenkinsci/plugins/saml/SamlAuthenticationToken.java

```diff
@@ -17,22 +17,23 @@
 
 package org.jenkinsci.plugins.saml;
 
-import org.acegisecurity.GrantedAuthority;
 import org.acegisecurity.providers.AbstractAuthenticationToken;
 
 public class SamlAuthenticationToken extends AbstractAuthenticationToken {
 
   private static final long serialVersionUID = 1L;
```

Comment:
I think there are two reasons:
1) the class is serializable
2) Eclipse (and may be some other IDEs) adds this field automatically to serializable classes (or notify that it should be added, not sure)